config/jobs/kubernetes/kops/kops-periodics-grid.yaml

```diff
@@ -47189,10 +47189,10 @@ periodics:
     test.kops.k8s.io/distro: u2004
     test.kops.k8s.io/extra_flags: --override=cluster.spec.cloudControllerManager.cloudProvider=aws
     test.kops.k8s.io/feature_flags: EnableExternalCloudController,SpecOverrideFlag
-    test.kops.k8s.io/k8s_version: ''
+    test.kops.k8s.io/k8s_version: '1.19'
     test.kops.k8s.io/kops_version: ''
     test.kops.k8s.io/networking: ''
-    testgrid-dashboards: sig-cluster-lifecycle-kops, google-aws, kops-grid, kops-distro-u2004, kops-k8s-latest
+    testgrid-dashboards: sig-cluster-lifecycle-kops, google-aws, kops-grid, kops-distro-u2004, kops-k8s-1.19
```

Comment:
Should we also add it to the `sig-aws-cloud-provider-aws` dashboard?

Reply:
Good idea -- I'll do that in another PR though as it will require some additional wiring up

Reply:
Done in #20260 - thanks for the suggestion!